README.md

```diff
@@ -291,9 +291,10 @@ hubspot.subscriptions.get(opts, cb)
 ### Email Events
 
 ```javascript
-hubspot.campaigns.get(opts, cb)
-hubspot.campaigns.getOne(id, appId, cb)
-hubspot.campaigns.events(opts, cb)
+hubspot.campaigns.getById()
```

Comment:
should this be `getById(id)`

Reply:
Discussed IRL, this is confusing as the naming implies this function is used to fetch a particular record by `id` but in reality, it maps to `/email/public/v1/campaigns/by-id`, which returns `all email campaign IDs associated with the portal`. I would love to see this updated to something like `getIds` but since that's a breaking change, it can wait.